TrajectoryPlannerROS:
  max_vel_x: 0.3
  min_vel_x: 0.00
  max_vel_theta: 0.52
  min_in_place_vel_theta: 0.0

  acc_lim_theta: 3.2
  acc_lim_x: 2.5
  acc_lim_y: 2.5

  holonomic_robot: false

  path_distance_bias: 0.1

  xy_goal_tolerance: 0.3
  yaw_goal_tolerance: 0.4


